src/src/Controller/ContactController.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Attribute\Route;
+
+class ContactController extends AbstractController
+{
+    #[Route('/contact', name: 'app_contact')]
+    public function index(): Response
+    {
+        return $this->render('contact/index.html.twig', [
+            'controller_name' => 'ContactController',
```

Comment:
This variable is generated by the framework as demo code. It should be removed as you don't need it.